server/stream.go

```diff
@@ -978,7 +978,7 @@ func (mset *Stream) Snapshot(deadline time.Duration, checkMsgs, includeConsumers
 		o.writeState()
 	}
 
-	return store.Snapshot(deadline, checkMsgs, includeConsumers)
+	return store.Snapshot(deadline, includeConsumers, checkMsgs)
```

Comment:
argument order in stream.Snapshot is different from store.Snapshot, I left as is but can fix - probably should

Reply:
I fixed this as well later on IIRC.

Reply:
I will double check.